### PR TITLE
feat: ET-1428: Added support for optional Issue ID in commit message

### PR DESCRIPTION
**Pull request by mattsalt123**
- Added Config to turn issue ID functionality on and off
- Changed prompt to include issue ID - controlled by config
- Added functionality to enter issue ID on the CLI during the commit process - controlled by config
- Examples changed in all languages to include Issue ID

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,12 +75,14 @@ oco
...
 ```env
-OCO_DESCRIPTION=<postface a message with ~3 sentences description of the changes>
+OCO_DESCRIPTION=<boolean, postface a message with ~3 sentences description of the changes>
 OCO_EMOJI=<boolean, add GitMoji>
 OCO_LANGUAGE=<locale, scroll to the bottom to see options>
 OCO_MESSAGE_TEMPLATE_PLACEHOLDER=<message template placeholder, default: '$msg'>
 OCO_PROMPT_MODULE=<either conventional-commit or @commitlint, default: conventional-commit>
 OCO_OPENAI_MAX_TOKENS=<max response tokens (default: 500)>
+OCO_ISSUE_ID=<boolean, issue ID included within commit message>
+OCO_ISSUE_ID_PREFIX=<optional prefix for issue ID, eg. 'ABC-'>
```
Comment (covolution) [inline README.md]: If the PREFIX is set then probably ISSUE_ID should be set to true

File: README.md [hunk 2]
```diff
@@ -75,12 +75,14 @@ oco
...
 
 ```env
-OCO_DESCRIPTION=<postface a message with ~3 sentences description of the changes>
+OCO_DESCRIPTION=<boolean, postface a message with ~3 sentences description of the changes>
 OCO_EMOJI=<boolean, add GitMoji>
 OCO_LANGUAGE=<locale, scroll to the bottom to see options>
 OCO_MESSAGE_TEMPLATE_PLACEHOLDER=<message template placeholder, default: '$msg'>
 OCO_PROMPT_MODULE=<either conventional-commit or @commitlint, default: conventional-commit>
 OCO_OPENAI_MAX_TOKENS=<max response tokens (default: 500)>
+OCO_ISSUE_ID=<boolean, issue ID included within commit message>
```
Comment (covolution) [inline README.md]: Not sure about `OCO_ISSUE_ID` as a name of a boolean?

**Review by covolution — CHANGES_REQUESTED** [review]

File: src/commands/config.ts [hunk 3]
```diff
@@ -26,8 +26,8 @@ export enum CONFIG_KEYS {
   OCO_MESSAGE_TEMPLATE_PLACEHOLDER = 'OCO_MESSAGE_TEMPLATE_PLACEHOLDER',
   OCO_PROMPT_MODULE = 'OCO_PROMPT_MODULE',
   OCO_AZURE_API_VERSION = 'OCO_AZURE_API_VERSION',
-  OCO_ISSUE_ID = 'OCO_ISSUE_ID',
-  OCO_ISSUE_ID_PREFIX = 'OCO_ISSUE_ID_PREFIX'
+  OCO_ISSUE_ENABLED= 'OCO_ISSUE_ENABLED',
```
Comment (vestron286) [inline src/commands/config.ts]: Add a space here after "ENABLED"

**Review by vestron286 — DISMISSED** [review]
Looks good! Ready to merge if everyone else is happy

**Review by danwill89 — DISMISSED** [review]

**Review by covolution — APPROVED** [review]

**Review by danwill89 — APPROVED** [review]

**Pull request merged**